pkgs/tools/misc/octosql/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildGo118Module
+, fetchFromGitHub
+}:
+
+with lib;
+
+buildGo118Module rec {
```

Comment:
nixpkgs uses buildGo118Module by default
```suggestion
, buildGoModule
, fetchFromGitHub
}:

buildGoModule rec {
```

Reply:
Thank you for pointing out!